libs/api/ebpf_api.cpp

```diff
@@ -534,7 +519,10 @@ ebpf_map_delete_element(fd_t map_fd, _In_ const void* key)
     if (result != EBPF_SUCCESS) {
         goto Exit;
     }
-    assert(key_size != 0);
+    if (key_size == 0) {
```

Comment:
I think for `BPF_MAP_TYPE_QUEUE`, key_size can be 0?

Reply:
Yes, but this API isn't valid for queue or stack.